client/components/data/query-site-checklist/index.js

```diff
@@ -18,7 +18,9 @@ export default function QuerySiteChecklist( { siteId } ) {
 	);
 
 	useEffect( () => {
-		dispatch( requestSiteChecklist( siteId, isSiteEligibleForFSE ) );
+		if ( siteId ) {
+			dispatch( requestSiteChecklist( siteId, isSiteEligibleForFSE ) );
```

Comment:
I wasn't able to find any usages of the query component that use `isSiteEligibleForFSE`. It's okay if it's undefined, as this defaults to false.

We can also drop the isRequired in propTypes

Reply:
Done.

Reply:
Done.